zephyr/module.yml

```
@@ -30,3 +30,7 @@ name: openthread
 build:
   cmake-ext: True
   kconfig-ext: True
+security:
+  external-references:
+    - cpe:2.3:o:google:openthread:2019-12-13:*:*:*:*:*:*:*
+    - pkg:github/openthread/openthread
```

should it be like this instead? feels wrong to put an old version of the CPE, but I also understand that the "new" one probably doesn't "exist" until a CVE is reported against it? not sure if it's ok to basically guess? or maybe just put a wildcard even if that means people might get false positives?)

re `pkg`, I think the latest released version that our fork is based off of should be there too but again, not sure

```suggestion
    - cpe:2.3:o:google:openthread:2023-07-06:*:*:*:*:*:*:*
    - pkg:github/openthread/openthread@thread-reference-20230706
```

---

Good question, I will ask around and see if I can find a good answer...

Like you say it doesn't seem right to either use the wrong version number, or to use a CPE that doesn't "exist" yet

---

Updated with your suggestion: the consensus internally is that it's better to specify the correct version number, and downstream users of the info can decide how to handle this if a CPE with this specific version does not exist yet.